sdks/go/test/integration/xlang/expansion_test.go

```diff
@@ -65,7 +48,13 @@ func TestAutomatedExpansionService(t *testing.T) {
 		t.Errorf("failed to start expansion service JAR, got %v", err)
 	}
 
-	checkPort(t, expansionPort, 15*time.Second)
+	ctx, canFunc := context.WithTimeout(context.Background(), 15*time.Second)
+	t.Cleanup(func() { canFunc() })
+	conn, err := grpc.DialContext(ctx, "localhost:"+expansionPort, grpc.WithInsecure())
```

Comment:
Still missed the WithBlock() call. It's currently always going to return immediately and pass.

Reply:
FIxed